tutorials/fit/NumericalMinimization.py

```diff
@@ -0,0 +1,85 @@
+#  \file
+#  \ingroup tutorial_fit
+#  \notebook -nodraw
+#  Example on how to use the new Minimizer class in ROOT
+#   Show usage with all the possible minimizers.
+#  Minimize the Rosenbrock function (a 2D -function)
+#  This example is described also in
+#  http://root.cern.ch/drupal/content/numerical-minimization#multidim_minim
+#  input : minimizer name + algorithm name
+#  randomSeed: = <0 : fixed value: 0 random with seed 0; >0 random with given seed
+#
+#  \macro_code
+#
+#  \author Lorenzo Moneta
+
+import ROOT
```

Comment:
This is a comment valid for the whole file. It would be better to stick to PEP8 as much as possible when writing Python code. For example, naming of variables and functions should follow snake_case and not CamelCase (apart from the calls to ROOT API which uses CamelCase).